### PR TITLE
Reviewed and edited the jails chapter for new UI.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/jails.rst [hunk 1]
```diff
@@ -1057,7 +1022,7 @@ Jail Software
 
 A jail is created with no software aside from the core packages
 installed as part of the selected version of FreeBSD. Software in a
-jail is managed by navigating to the :guilabel:`Shell` and logging
+jail is managed by going to the :guilabel:`Shell` and logging
```
Comment (wblock) [inline userguide/jails.rst]: Should be :menuselection:

File: userguide/jails.rst [hunk 2]
```diff
@@ -1057,7 +1022,7 @@ Jail Software
...
 into the jail with :command:`iocage console`. In this example,
```
Comment (wblock) [inline userguide/jails.rst]: But "iocage console" does not log in to the jail, it requires a jailname.  Use the :samp: command from above.

File: userguide/jails.rst [hunk 3]
```diff
@@ -1089,7 +1054,7 @@ the user has logged into *testjail01*:
 
 
 .. tip:: See :ref:`Using iocage` for more details about different
-   :command:`iocage` commands simple jail manipulation.
+   :command:`iocage` commands.
 
 The next sections detail two different options to install software
```
Comment (wblock) [inline userguide/jails.rst]: These intros at the end of sections are not useful, remove them.

File: userguide/jails.rst [hunk 4]
```diff
@@ -1089,7 +1054,7 @@ the user has logged into *testjail01*:
...
 inside a jail using :command:`pkg` or compiling the port directly.
```
Comment (wblock) [inline userguide/jails.rst]: s/using/with/
s/ directly//

File: userguide/jails.rst [hunk 5]
```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The quickest and easiest way to install software inside the jail is to
-install a FreeBSD package. FreeBSD packages are pre-compiled.  They
-contains all the binaries and a list of dependencies required for the
+install a FreeBSD package. FreeBSD packages are pre-compiled. They
```
Comment (wblock) [inline userguide/jails.rst]: install a FreeBSD package. Packages are precompiled and

File: userguide/jails.rst [hunk 6]
```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
...
+contain all the binaries and a list of dependencies required for the
 software to run on a FreeBSD system.
 
-A huge amount of software has been ported to FreeBSD, currently over
-24,000 applications, and most of that software is available as a
-package. One way to find FreeBSD software is to use the search bar at
+A huge amount of software is ported to FreeBSD. Most of that
+software is available as a package. One way to find FreeBSD software is
```
Comment (wblock) [inline userguide/jails.rst]: s/a package/packages/

File: userguide/jails.rst [hunk 7]
```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
...
 :command:`pkg install` command to install it. For example, to install
-the audiotag package, use this command:
-
-.. code-block:: none
-
-   pkg install audiotag
-
+the audiotag package, use the command :command:`pkg install audiotag`
 
 When prompted, type **y** to complete the installation. The
```
Comment (wblock) [inline userguide/jails.rst]: When prompted, press :kbd:`y` to complete the installation.

File: userguide/jails.rst [hunk 8]
```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
...
-installation messages will indicate if the package and its
+installation messages indicate if the package and
```
Comment (wblock) [inline userguide/jails.rst]: Messages will show the download and installation status.

File: userguide/jails.rst [hunk 9]
```diff
@@ -1102,33 +1067,28 @@ Installing FreeBSD Packages
...
 dependencies successfully download and install.
 
 .. warning:: Some older versions of FreeBSD used package systems
```
Comment (wblock) [inline userguide/jails.rst]: Without specifics, this is useless to the reader.  The package changeover was so long ago that this is not needed any more anyway.

File: userguide/jails.rst [hunk 10]
```diff
@@ -1233,14 +1192,14 @@ shows the :guilabel:`Configuration Options` for audiotag.
 This port has five configurable options (DOCS, FLAC, ID3, MP4,
```
Comment (wblock) [inline userguide/jails.rst]: This port has five configurable options: *DOCS*, *FLAC*, *ID3*, *MP4*,

File: userguide/jails.rst [hunk 11]
```diff
@@ -1233,14 +1192,14 @@ shows the :guilabel:`Configuration Options` for audiotag.
...
 and VORBIS) and each option is enabled (on) by default.
```
Comment (wblock) [inline userguide/jails.rst]: and *VORBIS*. In this port, each option is enabled by default.

File: userguide/jails.rst [hunk 12]
```diff
@@ -1233,14 +1192,14 @@ shows the :guilabel:`Configuration Options` for audiotag.
...
 
-FreeBSD packages are always built using the default options. When
-compiling a port, those options are presented in a menu, allowing the
-default values to be changed.
+FreeBSD packages are always built using the default options. Those
```
Comment (wblock) [inline userguide/jails.rst]: This is confusingly said.  Packages use default options.  Ports let the user select options.

File: userguide/jails.rst [hunk 13]
```diff
@@ -1233,14 +1192,14 @@ shows the :guilabel:`Configuration Options` for audiotag.
...
+options are presented in a menu when compiling a port. The default
+values can be changed.
 
 The Ports Collection must be installed in a jail before ports can be
```
Comment (wblock) [inline userguide/jails.rst]: s/in a jail/in the jail/

File: userguide/jails.rst [hunk 14]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
 
 
 Use the arrow keys to select an option and press :kbd:`spacebar`
-to toggle the value. When all the values are as desired, press
-:kbd:`Enter`.  The port will begin to compile and install.
+to toggle the value. Press :kbd:`Enter` when satisfied with the jail
+options. The port begin to compile and install.
```
Comment (wblock) [inline userguide/jails.rst]: s/begin/begins/

File: userguide/jails.rst [hunk 15]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
...
 
-.. note:: The configuration screen will not be shown again, even
-   if the build is stopped and restarted. It can be redisplayed
-   by typing :command:`make config`.  Change the settings, then
-   rebuild with :command:`make clean install clean`.
+.. note:: The configuration screen is not shown again. Redisplay it by
```
Comment (wblock) [inline userguide/jails.rst]: Again, imperative.  But the user does not have to do this, so it should not sound imperative.

"After options have been set, the configuration screen is normally not shown again.
Use :command:`make config` to display the screen and change options before
rebuilding the port with :command:`make clean install clean`.

File: userguide/jails.rst [hunk 16]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
...
+configuration screens that are shown before compiling begins. It
+is a good idea to watch the compile until it finishes and the
 command prompt returns.
 
-When the port is installed, it is registered in the same package
-database that manages packages. The same :command:`pkg info` command
-can be used to determine what was installed, as described in the
-previous section.
-
+The port is registered in the same package database that manages
```
Comment (wblock) [inline userguide/jails.rst]: s/The port is/Installed orts are/

File: userguide/jails.rst [hunk 17]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
...
+packages when it is installed. The :command:`pkg info`
```
Comment (wblock) [inline userguide/jails.rst]: packages. :command:`pkg info` can be used to determine
which ports were installed.

(Although it's not that easy, really.)

File: userguide/jails.rst [hunk 18]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 After packages or ports are installed, they must be configured and
-started. If familiar with the software, look for the configuration
-file in :file:`/usr/local/etc` or a subdirectory of it. Many FreeBSD
-packages contain a sample configuration file as a reference. If
-unfamiliar with the software, spend some time reading the software
-documentation to learn which configuration options are available and
-which configuration files require editing.
+started. The configuration file is in :file:`/usr/local/etc` or a
```
Comment (wblock) [inline userguide/jails.rst]: s/The configuration file is in/Configuration files are usually in/

File: userguide/jails.rst [hunk 19]
```diff
@@ -1276,45 +1234,40 @@ is displayed:
...
 Most FreeBSD packages that contain a startable service include a
 startup script which is automatically installed to
-:file:`/usr/local/etc/rc.d/`. After the configuration is complete, the
-starting of the service can be tested by running the script with the
-:command:`onestart` option. As an example, if openvpn is installed
-into the jail, these commands run its startup script and verify that
-the service started:
+:file:`/usr/local/etc/rc.d/`. After the configuration is complete, test
+starting the service by running the script with the :command:`onestart`
+option. For example, openvpn is installed into the jail and these
```
Comment (wblock) [inline userguide/jails.rst]: For example, with openvpn installed in the jail, these

File: userguide/jails.rst [hunk 20]
```diff
@@ -1413,8 +1365,8 @@ using :command:`iocage`.
 :command:`iocage` has several options to help users:
 
 * There is built-in help displayed by entering
-  :samp:`iocage --help | less`. Each subcommand also has help,
-  displayed by giving the subcommand name followed by the
+  :samp:`iocage --help | less`. Each subcommand also has help.
+  Display them by giving the subcommand name followed by the
```
Comment (wblock) [inline userguide/jails.rst]: Not "them".

Display help by adding the :literal:`--help` flag after the subcommand name.

File: userguide/jails.rst [hunk 21]
```diff
@@ -1413,8 +1365,8 @@ using :command:`iocage`.
...
   :literal:`--help` flag. For example, help for the
   :command:`activate` subcommand displays with
```
Comment (wblock) [inline userguide/jails.rst]: For example, :command:`iocage activate --help` shows help for the :command:`activate` subcommand.

File: userguide/jails.rst [hunk 22]
```diff
@@ -1430,7 +1382,7 @@ Managing iocage Jails
 ~~~~~~~~~~~~~~~~~~~~~
 
 Creating a jail automatically starts the iocage configuration process
-for the %brand% system. Jail properties can also be specified with the
+for the %brand% system. Jail properties are also specified with the
```
Comment (wblock) [inline userguide/jails.rst]: The original tense was correct.  Jail properties are not mandatory, just can be included.

File: userguide/jails.rst [hunk 23]
```diff
@@ -1430,7 +1382,7 @@ Managing iocage Jails
...
 :command:`iocage create` command.
 
 In this example a new jail named *examplejail* is created. Additional
```
Comment (wblock) [inline userguide/jails.rst]: s/is/has been/

File: userguide/jails.rst [hunk 24]
```diff
@@ -1511,7 +1463,7 @@ To adjust the properties of a jail, use :command:`iocage set` and
 :command:`iocage get all`:
 
 .. tip:: This example shows an abbreviated list of the properties for
-   **examplejail**. The iocage manual page (:command:`man iocage`)
+   **examplejail**. The iocage manual page ( :command:`man iocage`)
```
Comment (wblock) [inline userguide/jails.rst]: No space.

File: userguide/jails.rst [hunk 25]
```diff
@@ -4,39 +4,41 @@
...
 and configurations within a jail are isolated from both the %brand%
 host operating system and any other jails running on that system.
-During creation, the :guilabel:`VNET` option can be checked to provide
-the jail with an independent networking stack. The jail can then do its
-own IP broadcasting, which is required by some applications.
+During creation, set the :guilabel:`VNET` option to provide
+the jail with an independent networking stack. This allows the IP of the
+jail to be broadcasted, which is required by some applications.
 
 Advanced users can also create custom templates to automate the
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply userguide/jails.rst]: Pinged Brandon.

File: userguide/jails.rst [hunk 26]
```diff
@@ -759,17 +731,18 @@ entries available for a jail.
...
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd&apropos=0&sektion=8>`__
+must be enabled to access a jail from another system using SSH.
+
+Access the shell of the jail in the %brand% UI to enable SSH. Log into
+the UI. Click
+:menuselection:`Jails -->` |ui-options| :menuselection:`--> Shell`
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply userguide/jails.rst]: In my opinion it renders quite nicely.

File: userguide/jails.rst [hunk 27]
```diff
@@ -799,27 +772,21 @@ the desired jail. A root shell will open as shown in this example:
...
+   :samp:`iocage console <Jail Name>`.
 
 
 Enable sshd:
 
 .. code-block:: none
 
    sysrc sshd_enable="YES"
-
-   No existing entry or already set to NO:
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply userguide/jails.rst]: I guess not. I've run the command and the only thing to show up is "sshd_enable: NO -> YES".

File: userguide/jails.rst [hunk 28]
```diff
@@ -913,23 +880,23 @@ is configured on the %brand% system.
...
-currently added and allows new additional storage to be added.
-
-Navigate :menuselection:`Jails`, click |ui-options| on the desired
-jail, then :guilabel:`Mount points`. In the
-Mount points section is a list of all of the currently defined mount
-points.
-
-To add storage, navigate to
-:menuselection:`Mount points` and click |ui-add|.
+stored on the %brand% system. For example, transmission is a
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply userguide/jails.rst]: I couldn't find a markup for it in [Sphinx Docs](http://www.sphinx-doc.org/en/1.5.1/markup/inline.html). Thus, I will just capitalise it.

File: userguide/jails.rst [hunk 29]
```diff
@@ -4,41 +4,40 @@
...
 Advanced users can also create custom templates to automate the
 creation of pre-installed and customized operating systems.
 
-The ability to create multiple jails offers great flexibility
+The ability to create multiple jails offers flexibility
 regarding software management. For example, an administrator can
 choose to provide application separation by installing different
 applications in each jail, to create one jail for all installed
 applications, or to mix and match how software is installed into each
 jail.
```
Comment (wblock) [inline userguide/jails.rst]: If we can find an appropriate link, let's add:

.. note:: Some distributions of Linux can run in a jail. See
   `Linux in a FreeBSD jail <...>`__.

File: userguide/jails.rst [hunk 30]
```diff
@@ -53,6 +52,7 @@ This section will discuss:
 Jail Storage
 ------------
 
+
 Jails and FreeBSD releases are stored in a single dataset named
```
Comment (wblock) [inline userguide/jails.rst]: s/FreeBSD releases/download FreeBSD release files/

File: userguide/jails.rst [hunk 31]
```diff
@@ -82,7 +82,7 @@ Creating Jails
 %brand% has two options to create a jail. The :guilabel:`Jail Wizard`
 makes it easy to quickly create a jail. :guilabel:`Advanced Jail Creation`
 is the second method, where every possible jail option is configurable.
```
Comment (wblock) [inline userguide/jails.rst]: s/the second/an alternate/

File: userguide/jails.rst [hunk 32]
```diff
@@ -153,14 +152,13 @@ To open the full jail creation form, navigate
...
-properties as a simple method to quickly create an immediately usable
-jail. Many more advanced settings are available in the
-:guilabel:`Jail Properties`, :guilabel:`Network Properties`, and
-:guilabel:`Custom Properties` sections.
+A usable jail can be quickly created by setting only the required
+values, the :guilabel:`Jail Name` and :guilabel:`Release`. Additional
+settings are in the :guilabel:`Jail Properties`,
+:guilabel:`Network Properties`, and :guilabel:`Custom Properties`
+sections. :numref:`Table %s <jail_basic_props_tab>`
+summarizes the available options of the :guilabel:`Basic Properties` of
```
Comment (wblock) [inline userguide/jails.rst]: s/summarizes/shows/

File: userguide/jails.rst [hunk 33]
```diff
@@ -237,7 +235,7 @@ jail. Many more advanced settings are available in the
 
 Similar to the :ref:`Jail Wizard`, configuring the basic properties then
 clicking :guilabel:`Save` is often all that is needed to quickly create
-and begin using a new jail. To continue configuring more settings, click
+and use a jail. To continue configuring more settings, click
```
Comment (wblock) [inline userguide/jails.rst]: s/and use a/a new/

File: userguide/jails.rst [hunk 34]
```diff
@@ -307,30 +305,30 @@ these options.
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
    | exec.system_jail_user | string    | Set this boolean option to *True* to look for the :guilabel:`exec.jail_user` in the system                          |
```
Comment (wblock) [inline userguide/jails.rst]: I think the text should not use guilabel, but should just say "to look for the jail user in the system".

But it's not clear, please verify.

File: userguide/jails.rst [hunk 35]
```diff
@@ -307,30 +305,30 @@ these options.
...
+   |                       |           | `statfs(2) <https://www.freebsd.org/cgi/man.cgi?query=statfs>`__,                                                   |
+   |                       |           | `fstatfs(2) <https://www.freebsd.org/cgi/man.cgi?query=statfs>`__,                                                  |
+   |                       |           | `getfsstat(2) <https://www.freebsd.org/cgi/man.cgi?query=getfsstat>`__,                                             |
+   |                       |           | `fhstatfs(2) <https://www.freebsd.org/cgi/man.cgi?query=fhstatfs>`__,                                               |
    |                       |           | and other similar compatibility syscalls.                                                                           |
    |                       |           |                                                                                                                     |
    |                       |           | Set to *0*, all mount points are available without any restriction.                                                 |
```
Comment (wblock) [inline userguide/jails.rst]: All mount points are available without any restrictions if this is set to *0*.

File: userguide/jails.rst [hunk 36]
```diff
@@ -340,16 +338,16 @@ these options.
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
    | children.max          | integer   | The number of child jails allowed to be created by the jail or other jails under this jail. A limit of *0*          |
```
Comment (wblock) [inline userguide/jails.rst]: s/The number/Number/

File: userguide/jails.rst [hunk 37]
```diff
@@ -340,16 +338,16 @@ these options.
...
    |                       |           | restricts the jail from creating child jails. **Hierarchical Jails** in the `jail(8)                                |
-   |                       |           | <https://www.freebsd.org/cgi/man.cgi?query=jail&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports>`__      |
+   |                       |           | <https://www.freebsd.org/cgi/man.cgi?query=jail>`__                                                                 |
    |                       |           | man page explains the finer details.                                                                                |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
    | login_flags           | string    | The flags to pass to                                                                                                |
```
Comment (wblock) [inline userguide/jails.rst]: s/The flags/Flags/

File: userguide/jails.rst [hunk 38]
```diff
@@ -340,16 +338,16 @@ these options.
...
-   |                       |           | `login(1) <https://www.freebsd.org/cgi/man.cgi?query=login&manpath=FreeBSD+11.1-RELEASE+and+Ports>`__               |
+   |                       |           | `login(1) <https://www.freebsd.org/cgi/man.cgi?query=login>`__                                                      |
    |                       |           | when logging in to the jail using the **console** function.                                                         |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
-   | securelevel           | integer   | The value of the jail `securelevel <https://www.freebsd.org/doc/faq/security.html#idp60325448>`__ sysctl. A jail    |
+   | securelevel           | integer   | The value of the jail `securelevel <https://www.freebsd.org/doc/faq/security.html>`__ sysctl. A jail                |
```
Comment (wblock) [inline userguide/jails.rst]: s/The value/Value/

File: userguide/jails.rst [hunk 39]
```diff
@@ -340,16 +338,16 @@ these options.
...
    |                       |           | never has a lower securelevel than the host system. Setting this parameter allows a higher securelevel. If the host |
    |                       |           | system securelevel is changed, jail securelevel will be at least as secure.                                         |
    |                       |           | Securelevel options are 3, 2, 1, 0, and -1.                                                                         |
```
Comment (wblock) [inline userguide/jails.rst]: Put single stars around each allowed value.

File: userguide/jails.rst [hunk 40]
```diff
@@ -390,10 +388,10 @@ these options.
...
+   |                       |           | `traceroute(8) <https://www.freebsd.org/cgi/man.cgi?query=traceroute>`__                                            |
    |                       |           | require raw sockets to operate inside a jail. When set, the source IP addresses are enforced to comply with the     |
-   |                       |           | IP address bound to the jail, ignoring the the IP_HDRINCL flag on the socket.                                       |
+   |                       |           | IP address bound to the jail, ignoring the IP_HDRINCL flag on the socket.                                           |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
    | allow.chflags         | checkbox  | Set to treat jail users as privileged and allow the manipulation of system file flags. **securelevel** constraints  |
```
Comment (wblock) [inline userguide/jails.rst]: Most of these do not need the "Set", which is implied by the checkbox.  Just "Treat jail users" or "Allow privileged users" and so on.

File: userguide/jails.rst [hunk 41]
```diff
@@ -576,7 +574,7 @@ The final set of jail properties are contained in the
    +---------------------+-----------+---------------------------------------------------------------------------------------------------------------+
 
 
-Click :guilabel:`Save` when satisfied with all the different jail
+Click :guilabel:`Save` when satisfied with the different jail
```
Comment (wblock) [inline userguide/jails.rst]: Click :guilabel:`Save` when the desired jail properties have been set.

File: userguide/jails.rst [hunk 42]
```diff
@@ -742,15 +705,20 @@ entries available for a jail.
...
+   a stopped jail does not have a Stop option and cannot be accessed
+   with the :ref:`Shell`.
+
+
+.. note:: Modify the IP address information for a jail by using
    |ui-options| :guilabel:`Edit` instead of issuing the networking
-   commands directly from the command line of the jail. This ensures
-   the changes will survive a jail or %brand% reboot.
+   commands directly from the command line of the jail. This
+   ensures the changes survive a jail or %brand% reboot.
```
Comment (wblock) [inline userguide/jails.rst]: ensures the changes are saved and will survive a jail or %brand% reboot.

File: userguide/jails.rst [hunk 43]
```diff
@@ -759,17 +727,18 @@ entries available for a jail.
...
-in a jail, create a user account to log in with, and ssh access to
-the jail via any computer that has network access to the %brand% server
-is now possible.
 
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd>`__
+must be enabled to access a jail from another system using SSH.
```
Comment (wblock) [inline userguide/jails.rst]: This is a little confusing.  I think it means that sshd must be running inside that jail.

must be enabled in a jail to allow SSH access to that jail from another system.

File: userguide/jails.rst [hunk 44]
```diff
@@ -759,17 +727,18 @@ entries available for a jail.
...
+
+Access the shell of the jail in the %brand% UI to enable SSH. Log into
```
Comment (wblock) [inline userguide/jails.rst]: This sentence is redundant, it's just stating a part of what is about to be done below.  Better without it.

File: userguide/jails.rst [hunk 45]
```diff
@@ -759,17 +727,18 @@ entries available for a jail.
...
+the UI. Click
+:menuselection:`Jails -->` |ui-options| :menuselection:`--> Shell`
+for the desired jail. The jail status must be :guilabel:`up` for
+:guilabel:`Shell` to be available. Start the jail by clicking
+|ui-options| and :guilabel:`Start`. The jail root shell is shown in this
+example:
```
Comment (wblock) [inline userguide/jails.rst]: The ordering of sentences in this paragraph is weird.  It starts with telling the user to open a shell, then tells them the jail must be up, then tells them to start the jail.  This needs to be simplified.
1. The jail must be up ("running"?  whatever the exact thing shown in the UI says).
2. If the jail is not up, start it by doing this.
3. Click this to start a shell on the jail.

File: userguide/jails.rst [hunk 46]
```diff
@@ -1102,33 +1059,21 @@ Installing FreeBSD Packages
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The quickest and easiest way to install software inside the jail is to
-install a FreeBSD package. FreeBSD packages are pre-compiled.  They
-contains all the binaries and a list of dependencies required for the
+install a FreeBSD package. FreeBSD packages are precompiled and they
+contain all the binaries and a list of dependencies required for the
```
Comment (wblock) [inline userguide/jails.rst]: s/they//

File: userguide/jails.rst [hunk 47]
```diff
@@ -1102,33 +1059,21 @@ Installing FreeBSD Packages
...
 software to run on a FreeBSD system.
 
-A huge amount of software has been ported to FreeBSD, currently over
-24,000 applications, and most of that software is available as a
-package. One way to find FreeBSD software is to use the search bar at
+A huge amount of software is ported to FreeBSD. Most of that
```
Comment (wblock) [inline userguide/jails.rst]: This was correct with "has been".  If I changed that, sorry.

File: userguide/jails.rst [hunk 48]
```diff
@@ -1230,17 +1174,15 @@ shows the :guilabel:`Configuration Options` for audiotag.
    Configuration Options for Audiotag
 
 
-This port has five configurable options (DOCS, FLAC, ID3, MP4,
-and VORBIS) and each option is enabled (on) by default.
+This port has five configurable options: *DOCS*, *FLAC*, *ID3*, *MP4*,
+and *VORBIS*. In this port, each option is enabled by defualt.
```
Comment (wblock) [inline userguide/jails.rst]: Typo.  Also, "In this port" is redundant.  Maybe

"Stars (:literal:`*`) show which options are enabled."

File: userguide/jails.rst [hunk 49]
```diff
@@ -1276,45 +1217,41 @@ is displayed:
...
+options. The port begins to compile and install.
 
-.. note:: The configuration screen will not be shown again, even
-   if the build is stopped and restarted. It can be redisplayed
-   by typing :command:`make config`.  Change the settings, then
-   rebuild with :command:`make clean install clean`.
+.. note:: After options have been set, the configuration screen is
+   normally not shown again. Use :command:`make config` to display the
+   screen and change options before rebuilding the port with
+   :command:`make clean` install clean.
```
Comment (wblock) [inline userguide/jails.rst]: The final backtick is after the final "clean":
:command:`make clean install clean`.

File: userguide/jails.rst (hunk 29 above)
Comment (a-stjohn) [reply userguide/jails.rst]: In progress.

File: userguide/jails.rst (hunk 30 above)
Comment (a-stjohn) [reply userguide/jails.rst]: Did you mean downloaded* FreeBSD release files?

File: userguide/jails.rst (hunk 34 above)
Comment (a-stjohn) [reply userguide/jails.rst]: So it is an actual field in the jail properties tab of "Advanced Jail Creation" on the FreeNAS system. How would you like to go about it?

File: userguide/jails.rst [hunk 50]
```diff
@@ -257,15 +256,15 @@ these options.
    |                       |           |                                                                                                                     |
    +=======================+===========+=====================================================================================================================+
    | devfs_ruleset         | integer   | The number of the `devfs(8)                                                                                         |
```
Comment (wblock) [inline userguide/jails.rst]: s/The number/Number/

File: userguide/jails.rst [hunk 51]
```diff
@@ -187,19 +186,19 @@ jail. Many more advanced settings are available in the
    |                           |                |                                                                                                         |
    +---------------------------+----------------+---------------------------------------------------------------------------------------------------------+
    | DHCP autoconfigure IPv4   | checkbox       | Set to automatically configure IPv4 networking with an independent VNET stack. :guilabel:`VNET` and     |
```
Comment (wblock) [inline userguide/jails.rst]: "Set to" not needed on these.

File: userguide/jails.rst [hunk 52]
```diff
@@ -389,56 +388,56 @@ these options.
    |                       |           | instead.                                                                                                            |
    |                       |           |                                                                                                                     |
    +-----------------------+-----------+---------------------------------------------------------------------------------------------------------------------+
-   | allow.raw_sockets     | checkbox  | Set to allow raw sockets. Utilities like                                                                            |
-   |                       |           | `ping(8) <https://www.freebsd.org/cgi/man.cgi?query=ping&manpath=FreeBSD+11.1-RELEASE+and+Ports>`__ and             |
-   |                       |           | `traceroute(8) <https://www.freebsd.org/cgi/man.cgi?query=traceroute&manpath=FreeBSD+11.1-RELEASE+and+Ports>`__     |
+   | allow.raw_sockets     | checkbox  | Allows raw sockets. Utilities like                                                                                  |
```
Comment (wblock) [inline userguide/jails.rst]: s/Allows/Allow/

File: userguide/jails.rst [hunk 53]
```diff
@@ -548,12 +547,12 @@ The final set of jail properties are contained in the
    |                     |           |                                                                                                               |
    +---------------------+-----------+---------------------------------------------------------------------------------------------------------------+
    | mount.procfs        | checkbox  | Set to allow mounting of a                                                                                    |
```
Comment (wblock) [inline userguide/jails.rst]: Remove "Set to" as above.

File: userguide/jails.rst [hunk 54]
```diff
@@ -759,17 +728,20 @@ entries available for a jail.
...
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd>`__
+must be enabled in a jail to allow SSH access to that jail from another
+system.
+
+The jail :guilabel:`Status` must be :guilabel:`up`
+before the :guilabel:`Shell` option is available. Start the jail by
```
Comment (wblock) [inline userguide/jails.rst]: Better, but needs "If the jail is not up, start it by..."

File: userguide/jails.rst [hunk 55]
```diff
@@ -759,17 +728,20 @@ entries available for a jail.
...
 
-Log in to a shell to get :guilabel:`cli` access to the jail to enable SSH
-on. To use the Jails Shell click |ui-options| --> :guilabel:`Shell` on
-the desired jail. A root shell will open as shown in this example:
+The ssh daemon
+`sshd(8) <https://www.freebsd.org/cgi/man.cgi?query=sshd>`__
+must be enabled in a jail to allow SSH access to that jail from another
+system.
+
+The jail :guilabel:`Status` must be :guilabel:`up`
```
Comment (wblock) [inline userguide/jails.rst]: No :guilabel: on the up, it's not a control.

**Pull request merged**